lizmap/modules/lizmap/lib/Form/QgisForm.php

```diff
@@ -183,6 +183,8 @@ public function __construct($layer, $form, $featureId, $loginFilteredOverride, A
 
         // Set form's private data
         $privateData = array();
+
+        // FIXME why ? there is already these fields into the form. We duplicate data here
```

Comment:
If data are duplicated here, we have to update:
* `qgisForm.listener.php` that uses them for checking if the jForm is a qgisForm https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/lizmap/classes/qgisForm.listener.php#L21
* `qgisExpressionUtils.class.php` that uses them for getting lizmap project from form https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/lizmap/classes/qgisExpressionUtils.class.php#L285
* `controllers/edition.classic.php` that uses them for building a qgisForm to evaluate input fields group visibilities https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/lizmap/controllers/edition.classic.php#L1456

Reply:
I removed all this private data excepts `['qgis_control'][X]['valueRelationData']` and `qgis_groupDependencies`